README.md

```diff
@@ -2,6 +2,14 @@
 [![GitHub license](https://img.shields.io/badge/license-LGPL-blue.svg)](https://raw.githubusercontent.com/PedestrianDynamics/jupedsim/master/LICENSE)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/jupedsim)
 ![PyPI - Version](https://img.shields.io/pypi/v/jupedsim)
+[![Build CI wheels](https://github.com/PedestrianDynamics/jupedsim/actions/workflows/cibuildwheel.yml/badge.svg)](https://github.com/PedestrianDynamics/jupedsim/actions/workflows/cibuildwheel.yml)
+[![Documentation Status](https://img.shields.io/badge/docs-up--to--date-brightgreen)](https://jupedsim.org/stable)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/9491/badge)](https://www.bestpractices.dev/projects/9491)
```

Comment:
I would like to see a few changes to the questionnaire on openSSF.

Reply:
In Section Basics:

**Issue 1**

When talking about the project website our jupedsim.org presence and the GitHub presence are used interchangeably. I think this is not the way this was intended to. Right now the information is not as easy accessible as I think the questionnaire wants it to be.

Looking at it very strictly we should put it to unmet or update the doc on Jupedsim 

**Issue 2**

We should file a bug with the OSI website to include the `LGPL-3.0-or-later` SPDX identifier in their website as OSI approved. At least its listed as OSI approved on https://spdx.org/licenses/

Then at least we give something back to them (found a bug) or we learned that it is actually not approved, which would be a surprise.

Reply:
In section Analysis:

We have to set everything to unmet as we do not have any automated static or dynamic code analysis running.

Reply:
In section Quality:

* The software is buildable only with FLOSS tools.
* We DO have an automated test suite.
* The test are invocable in a standard way for that language
* We DO have a working CI that executes the tests

Regarding the other points, yep we are lacking :/